nucleus/parent/pom.xml

```diff
@@ -99,11 +99,11 @@
 
         <!-- Jakarta XML Binding -->
         <jakarta.xml.bind-api.version>4.0.0</jakarta.xml.bind-api.version>
-        <jakarta.jaxb-impl.version>4.0.0</jakarta.jaxb-impl.version>
```

Comment:
:+1: for the jaxb-impl upgrade so we can validate that it solves the https://github.com/eclipse-ee4j/jaxb-tck/issues/78 (w2.org http redirect to https) for GlassFish and so we can update https://github.com/eclipse-ee4j/jakartaee-tck/wiki/Jakarta-EE-10.0-TCK-results as well.